pkg/events/events.go

```diff
@@ -123,6 +123,11 @@ func New(object interface{}, eventType config.EventType, resource, clusterName s
 		event.Count = eventObj.Count
 		event.Action = eventObj.Action
 		event.TimeStamp = eventObj.LastTimestamp.Time
+		//Compatible with events.k8s.io/v1
```

Comment:
```suggestion
		// Compatible with events.k8s.io/v1
```

Reply:
@Lion916 can be merged once this minor comment is addressed

Reply:
Also please rebase with the latest develop to fix the CI failure

Reply:
@PrasadG193 Could you help to review this PR? Thanks~

Reply:
Thanks for being patient. :pray: